internal/operator/clusterutilities.go

```diff
@@ -735,6 +737,42 @@ func GetPgbackrestS3EnvVars(cluster crv1.Pgcluster, clientset *kubernetes.Client
 	} else {
 		s3EnvVars.PgbackrestS3Region = Pgo.Cluster.BackrestS3Region
 	}
+	if cluster.Spec.BackrestS3URIStyle != "" {
+		s3EnvVars.PgbackrestS3URIStyle = cluster.Spec.BackrestS3URIStyle
+	} else {
+		s3EnvVars.PgbackrestS3URIStyle = Pgo.Cluster.BackrestS3URIStyle
+	}
+	if cluster.Spec.BackrestS3VerifyTLS != "" {
+		s3EnvVars.PgbackrestS3VerifyTLS = cluster.Spec.BackrestS3VerifyTLS
+	} else {
+		s3EnvVars.PgbackrestS3VerifyTLS = Pgo.Cluster.BackrestS3VerifyTLS
+	}
+
+	// if set, pgBackRest URI style must be set to either 'path' or 'host'. If it is neither,
+	// log an error and stop the cluster from being created.
+	if s3EnvVars.PgbackrestS3URIStyle != "path" && s3EnvVars.PgbackrestS3URIStyle != "host" &&
+		s3EnvVars.PgbackrestS3URIStyle != "" {
+		log.Error("pgBackRest S3 URI style must be set to either \"path\" or \"host\".")
+		return ""
+	}
+
+	// If the pgcluster has already been set, either by the PGO client or from the
+	// CRD definition, parse the boolean value given.
+	// If this value is not set, then parse the value stored in the default
+	// configuration and set the value accordingly
+	verifyTLS, _ := strconv.ParseBool(Pgo.Cluster.BackrestS3VerifyTLS)
+
+	if cluster.Spec.BackrestS3VerifyTLS != "" {
+		verifyTLS, _ = strconv.ParseBool(cluster.Spec.BackrestS3VerifyTLS)
+	}
+
+	// Now, assign the expected value for use by pgBackRest, in this case either 'y'
+	// to enable or 'n' to disable TLS verification.
+	if verifyTLS {
+		s3EnvVars.PgbackrestS3VerifyTLS = "y"
+	} else {
+		s3EnvVars.PgbackrestS3VerifyTLS = "n"
+	}
```

Comment:
One more suggestion, in the guise of the above changes :smile:

```suggestion
	s3EnvVars.PgbackrestS3VerifyTLS = "n"
	
	if verifyTLS {
		s3EnvVars.PgbackrestS3VerifyTLS = "y"
	}
```

Reply:
Sure thing, updated.